fem/transfer.cpp

```diff
@@ -912,8 +913,11 @@ TransferOperator::TransferOperator(const FiniteElementSpace& lFESpace_,
    }
    else if (lFESpace_.GetMesh()->GetNE() > 0
             && hFESpace_.GetMesh()->GetNE() > 0
+            && lFESpace_.GetVDim() == 1
```

Comment:
Why are these new checks for `vdim` necessary? Also, how can `GetNE() > 0` not be true in the existing lines above?

Reply:
The checks for the number of elements were already there ... the new check on the `vdim` is added because it seems that the `TensorProductPRefinementOperator` does not support `vdim>1`

Reply:
I wonder what could possibly break if `GetNE() > 0` is removed. We can leave it, but I'm just curious.

Reply:
Maybe this is to capture the case of an empty MPI rank (for the `ParMesh` case)